public/css/style.css

```diff
@@ -0,0 +1,17 @@
+/* Overide of Tachyon classes */
+
+.bg-dark-blue {
+    background-color: #345C7E;
+}
+
+.bg-light-blue {
+    background-color: #5FBAC0;
+}
+
+.shadow-1 {
+    box-shadow: 0px 4px 4px 0px rgba(0,0,0,0.25);
+} 
+
+h1 {
```

Comment:
this one's not an override so just make that clear with a comment or something :)